src/index.mako.html

```diff
@@ -726,22 +726,34 @@
         var cacheAdd = '${version}' != '' ? '/' + '${version}' : '';
         var pathname = location.pathname.replace(/(index|mobile|embed)\.html$/g, '');
 
+        function getParam(name){
+          return decodeURIComponent((new RegExp('[?|&]' + name + '=' + '([^&;]+?)(&|#|;|$)').exec(location.search)||[,""])[1].replace(/\+/g, '%20'))||null;
```

Comment:
the regexp helps us find the parameter value. What is wrong with it? Is there another method you prefer I'd use?